src/PeliasMapkitExtensions.swift

```diff
@@ -144,3 +144,10 @@ extension SearchBoundaryRect {
     self.minLatLong = GeoPoint(latitude: maxCoordinate.latitude, longitude: minCoordinate.longitude)
   }
 }
+
+public extension GeoPoint {
```

Comment:
Opened an issue to track: https://github.com/mapzen/ios/issues/141

Feel free to post additional thoughts there.